app/data/generators/events.js

```diff
@@ -1,14 +1,25 @@
-module.exports = (faker, params) => {
+const { DateTime } = require('luxon')
+const faker = require('faker')
+faker.locale = 'en_GB'
+
+function getRandomDate(date, min = 1, max = 20) {
```

Comment:
Is this any random date or a random date which is in past the date that's passed in?

`getFutureDate(date, limit = 20)` might work better here?

Reply:
This is a random future date from the date passed into the function. So it's better to name the function as you suggest `getFutureDate()`